limiter/limiter.go

```diff
@@ -163,6 +166,23 @@ func (l *Limiter) GetMax() int64 {
 	return l.max
 }
 
+// SetBurst is thread-safe way of setting maximum burst size.
+func (l *Limiter) SetBurst(burst int) *Limiter {
+	l.Lock()
+	l.burst = burst
+	l.Unlock()
+
+	return l
+}
+
+// GetBurst is thread-safe way of setting maximum burst size.
+func (l *Limiter) GetBurst() int {
+	l.RLock()
+	l.RUnlock()
```

Comment:
Was lurking around your PR cause I am looking forward to this PR.

Seems like you are missing a `defer` statement here

Reply:
thanks! I just fixed it